packages/ag-charts-community/src/chart/overlay/overlay.ts

```diff
@@ -44,7 +44,12 @@ export class Overlay extends BaseProperties {
             } else {
                 const tempDiv = createElement('div');
                 tempDiv.innerHTML = htmlContent;
-                this.content = tempDiv.firstElementChild! as HTMLElement;
+                const { firstElementChild } = tempDiv;
+                if (firstElementChild instanceof HTMLElement && tempDiv.childElementCount === 1) {
+                    this.content = firstElementChild;
+                } else {
+                    this.content = tempDiv;
```

Comment:
This might make it harder to predict what would happen - we could just log a warning instead that only the first element will get used?

Reply:
The renderer() functions are for more advanced use-cases, so I'm happy that there is some risk that users don't have guard-rails here.